### PR TITLE
feat(core): Show Public API key value only once (no-changelog)

### PR DESCRIPTION
**Pull request by RicardoE105**
## Summary

Users could get the Public API key value in the API Settings anytime. This is not a good practice. This PR redacts the API Key after creation.

Before:

https://www.loom.com/share/458e1a40e361402ebc1acc6c941423b8

Now:

https://www.loom.com/share/98f3d2e4dfe0483da7043d7d1955fa1a

## Related Linear tickets, Github issues, and Community forum posts

Idea came from the discussion in the ticket: https://linear.app/n8n/issue/SEC-55/n8n-public-api-key-security

## Review / Merge checklist

- [x] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [x] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

**Comment by RicardoE105** [comment]
Fixes SEC-56

File: packages/cli/test/unit/controllers/me.controller.test.ts [hunk 1]
```diff
@@ -223,7 +223,7 @@ describe('MeController', () => {
 	describe('API Key methods', () => {
```
Comment (RicardoE105) [inline packages/cli/test/unit/controllers/me.controller.test.ts:223]: Do not think this test suite is needed. We are testing this endpoints in `me.api.test.ts`

**Review by netroy — DISMISSED** [review]
🎉

**Review by netroy — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.53.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.53.0)

**Pull request merged**